src/ch20-01-single-threaded.md

```diff
@@ -2,7 +2,7 @@
 
 我們會先從建立一個可運作的單一執行緒網頁伺服器作為起始。在我們開始之前，讓我們快速瞭解一下建構網頁伺服器會涉及到的協定。這些協定的細節超出本書的範疇，不過以下簡短的概覽可以提供你所需要知道的訊息。
 
-網頁伺服器會涉及到的兩大協定為**超文本傳輸協定（Hypertext Transfer Protocol, HTTP）**與**傳輸控制協定（Transmission Control Protocol, TCP）**。這兩種協定都是**請求-回應（request-response）**協定，這意味著**客戶端**會初始一個請求，然後**伺服器**接聽請求並提供回應給客戶端。協定會定義這些請求與回應的內容。
+網頁伺服器會涉及到的兩大協定為**超文本傳輸協定（Hypertext Transfer Protocol, HTTP）**與**傳輸控制協定（Transmission Control Protocol, TCP）**。這兩種協定都是**請求 - 回應（request-response）**協定，這意味著**客戶端**會初始一個請求，然後**伺服器**接聽請求並提供回應給客戶端。協定會定義這些請求與回應的內容。
```

Comment:
I am not sure about when to use which dash. Found [this article](https://english.stackexchange.com/questions/28601/appropriate-dash-to-use-when-attributing-a-quotation/59320) though. Could you explain why this change?